ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -179,7 +182,7 @@ objects:
         # create cluster for test
         cd /go/src/github.com/openshift/openshift-azure/
         timeout 1h ./hack/create.sh ${RESOURCEGROUP}
-        if [[ -n "$TEST_IN_PRODUCTION" ]]; then
+        if [[ -n "$TEST_IN_PRODUCTION" ]] || [[ "${RUNNING_UNDER_TEST}" == "" ]]; then
           make e2e-prod
```

Comment:
Is this same? @kargakis ?  or we need 3 branches for tests?
E2E - FAKE RP NO RUT
E2E - FAKE RP RUT
E2E - REAL RP?

Reply:
I had the same question.

Reply:
@kargakis, if you get a second could you comment on this?

Reply:
This test only makes sense in prod cluster where end users don't have access to azure resources